internal/diff/printer.go

```diff
@@ -0,0 +1 @@
+package diff
```

Comment:
You should also add a todo here, because this empty file seems awkward

Reply:
This is a work in progress patchset. I think this won't be a problem as I will fill it later.